### PR TITLE
Update migrate-files-migration-manager.md

### PR DESCRIPTION
**Pull request by msbemba**
added information that currently this Migration Manager Google isn't available for any of the Government clouds.

### CODE REVIEW

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9912

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/moveto-microsoft-365/migrate-files-migration-manager.md [hunk 1]
```diff
@@ -45,6 +45,8 @@ When you move to Microsoft 365 for business from Google Workspace, you'll want t
 
 All the users should have signed in to Microsoft 365 for business and set up their OneDrive for Business. To do this, go to [office.com](https://office.com), sign in with your Microsoft 365 for business credentials, and then choose OneDrive.
 
+Currently this Migration Manager Google isn't available for any of the Government clouds.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/moveto-microsoft-365/migrate-files-migration-manager.md]:
```suggestion
> [!NOTE]
> Currently, Migration Manager isn't available for Google on any of the Government clouds.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@efrene Please merge this Pr. Thanks

**Comment by efrene** [comment]
Hi Yogesh,
I'm waiting to hear back from a PM on this to make sure it is true before merging.
Thanks,
Efren


Sent from Outlook<http://aka.ms/weboutlook>

________________________________

**Pull request merged**